DatabaseServer/moxa_data.py

```diff
@@ -34,27 +39,40 @@
 DELETE_PORTS = """
 DELETE FROM moxa_details.port_mappings;"""
 
-SYSTEM_MIBS = ["DISMAN-EXPRESSION-MIB::sysUpTimeInstance", "SNMPv2-MIB::sysName"]
+SYSTEM_MIBS = ["DISMAN-EXPRESSION-MIB::sysUpTimeInstance", "SNMPv2-MIB::sys_name"]
```

Comment:
I suspect this might have been a bad refactor / search-replace?

Or are you sure that this new mib path is correct?

Reply:
```suggestion
SYSTEM_MIBS = ["DISMAN-EXPRESSION-MIB::sysUpTimeInstance", "SNMPv2-MIB::sysName"]
```